x/gov/abci.go

```diff
@@ -71,10 +72,12 @@ func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) {
 			if err == nil {
 				for idx, msg = range messages {
 					handler := keeper.Router().Handler(msg)
```

Comment:
A defensive `nil` check on the handler here would probably be useful too. 

What if I tried to exec an osmosis `MsgSwap` or something on the hub. I'm assuming it should fail before ever getting here, but the code looks like it relies on the conversion to `Any` as an unregistered type URL to prevent this. 
I don't see any harm in writing this as:

```
if handler := keeper.Router().Handler(msg); handler != nil {
    res, err := handler(...)
...
}
```